internal/api/service/service_test.go

```diff
@@ -29,12 +31,28 @@ func newMockK8sClient(name string, time time.Time) *mockK8sClient {
 	}
 }
 
-func TestListReleasesPopulatesFromConfigMap(t *testing.T) {
-	name := "configmap-1"
+type mockGitClient struct {
+	Org    string
```

Comment:
Another red flag, these fields are never used, so why are they here in the first place? The `mockGitClient` should be using the `testify/mock.Mock` type for sure.